client/components/TestQueueCompletionStatusListItem/PreviouslyAutomatedTestCompletionStatus/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/client';
+import { TEST_PLAN_RUN_ASSERTION_RESULTS_QUERY } from '../queries';
+import { isAssertionValidated } from '../../../utils/automation';
+
+const PreviouslyAutomatedTestCompletionStatus = ({
+    runnableTestsLength,
+    testPlanRunId,
+    id
+}) => {
+    const { data: testPlanRunAssertionsQueryResult } = useQuery(
+        TEST_PLAN_RUN_ASSERTION_RESULTS_QUERY,
+        {
+            variables: {
+                testPlanRunId: testPlanRunId
+            },
+            fetchPolicy: 'cache-and-network'
+        }
+    );
+
+    const numValidatedTests = useMemo(() => {
+        if (!testPlanRunAssertionsQueryResult) {
+            return 0;
+        }
+
+        const { testPlanRun } = testPlanRunAssertionsQueryResult;
+
+        if (!testPlanRun) {
+            return 0;
+        }
+
+        const { testResults = [] } = testPlanRun;
+
+        return testResults.reduce((acc, testResult) => {
+            const isTestValidated = testResult.scenarioResults.every(
+                scenarioResult =>
+                    scenarioResult.assertionResults.every(isAssertionValidated)
+            );
+
+            return isTestValidated ? acc + 1 : acc;
+        }, 0);
+    }, [testPlanRunAssertionsQueryResult]);
+
+    return (
+        <div id={id} className="text-secondary">
+            {`${numValidatedTests} of ${runnableTestsLength} tests evaluated`}
+        </div>
+    );
+};
```

Comment:
This is not a bug but more an observation using this for the first time. There is a bunch of automation-related UI that appears when automation is running, but after automation is complete and the work has been assigned to someone, it becomes difficult to understand that the automated results are actually automated results.

<img width="1303" alt="Automated says 0 of 8 test evaluated and manual says 0 of 8 tests completed, but that's the only difference." src="https://github.com/w3c/aria-at-app/assets/5255858/d9147c77-1fa2-4638-a9a6-ec935fbc3aa5">

My original expectation was that the "Manage NVDA Bot Run" button and the UI listing the number of completed tests would remain permanently, but I can imagine that there are probably good reasons to eventually hide it.

My suggestion would be to keep an icon or some other reference to NVDA Bot around, to make it more clear that these are actually automated results.

Reply:
I think this is design feedback since what you are responding to is directly from a design that was created by isaac and iterated on based on feedback by matt. My sense is that there was an intentional decision to reduce the distinction as much as possible for how bot tests that have been reassigned are displayed.

Reply:
Archived this feedback in [this comment](https://bocoup.slack.com/archives/CB9ESG2AF/p1703199005107479) for possible use on further design cycles.